test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -192,6 +207,61 @@
       (let [resp (http-client/get "http://localhost:8180/alternate-status/v1/services")]
         (is (= 200 (:status resp)))))))
 
+(deftest auth-service-ssl-status-endpoint-test
+  (testing "with auth service running and cert auth enabled for endpoint"
+    (with-app-with-config
+      app
+      [jetty9-service/jetty9-service
+       webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
+       status-service
+       tk-auth/authorization-service]
+       (merge ssl-status-service-config
+         {:authorization {:version 1
+                          :rules [{:match-request
+                                   {:path "/status/v1/services"
+                                    :type "path"
+                                    :method "get"}
+                                   :sort-order 500
+                                   :allow "*"
+                                   :allow-unauthenticated false
+                                   :name "puppetlabs status service"}]}})
+        (testing "without authentication returns 403"
+          (let [resp (http-client/get "https://localhost:8180/status/v1/services" incomplete-ssl-config)]
+            (is (= 403 (:status resp)))))
```

Comment:
This causes an error message logged during the test run. Can we wrap this with a `with-test-logging` block to keep that from happening?